account_global_discount/static/description/index.html

```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
```

Comment:
This file is quite incomplete.

![image](https://user-images.githubusercontent.com/3088246/87423643-e8f2f380-c5da-11ea-8565-6ddada1d7f6a.png)

Reply:
Well, it will be generated on merge, although I usually pre-generate it, but not sure if this time it wasn't done.